examples/platforms/cc2538/platform-cc2538.h

```diff
@@ -90,4 +90,18 @@ void cc2538RandomInit(void);
  */
 void cc2538UartProcess(void);
 
+#if OPENTHREAD_CONFIG_CC2538_WITH_CC2592 && OPENTHREAD_CONFIG_CC2592_USE_HGM
+/**
+ * Change the state of the CC2592 HGM pin.
+ *
+ * @param	aState	Whether or not to enable HGM
+ */
+void cc2538RadioSetHgm(bool aState);
+
+/**
+ * Retrieve the state of the CC2592 HGM pin.
+ */
+bool cc2538RadioGetHgm();
```

Comment:
nit: "All void functions or methods shall explicitly declare and specify the void type keyword."